code/modules/research/techweb/all_nodes.dm

```diff
@@ -790,6 +790,15 @@
 	"phazon_peri", "phazon_targ", "phazon_armor")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 
+/datum/techweb_node/sidewinder
+	id = "mech_sidewinder"
+	display_name = "EXOSUIT: Sidewinder"
+	description = "Sidewinder exosuit designs"
+	prereq_ids = list("adv_mecha", "adv_weaponry", "mech_gygax")
```

Comment:
Why does it require gygax research? Also, gygax node already requires adv_mecha so it's redundant here

Reply:
It's a sort of sidegrade to the gygax, but ill get rid of the advanced mech requirement.

Reply:
I would think that a "sidegrade" would be equally as difficult to get, not strictly harder

Reply:
I like the sidegrade idea, you can't really have an alternate version if you never made the first one. But maybe make it 1.5k points instead?

Reply:
2500 is just the standard cost for regular nodes, I'd rather keep the mech designs at that price. I removed the advanced mecha node.